src/System.Text.Json/tests/Serialization/OptionsTests.cs

```diff
@@ -25,5 +49,69 @@ public static void DefaultBufferSize()
             options.DefaultBufferSize = 1;
             Assert.Equal(1, options.DefaultBufferSize);
         }
+
+        [Fact]
+        public static void AllowTrailingCommas()
+        {
+            Assert.Throws<JsonReaderException>(() => JsonSerializer.Parse<int[]>("[1,]"));
+
+            var options = new JsonSerializerOptions();
+            options.AllowTrailingCommas = true;
+
+            int[] value = JsonSerializer.Parse<int[]>("[1,]", options);
+            Assert.Equal(1, value[0]);
+        }
+
+        [Fact]
+        public static void WriteIndented()
+        {
+            var obj = new BasicCompany();
+            obj.Initialize();
+
+            // Verify default value.
+            string json = JsonSerializer.ToString(obj);
+            Assert.DoesNotContain("\r", json);
+
+            // Verify default value on options.
+            var options = new JsonSerializerOptions();
+            json = JsonSerializer.ToString(obj, options);
+            Assert.DoesNotContain("\r", json);
+
+            // Change the value on options.
+            options = new JsonSerializerOptions();
+            options.WriteIndented = true;
+            json = JsonSerializer.ToString(obj, options);
+            Assert.Contains("\r", json);
```

Comment:
I am surprised this is passing on unix.

Reply:
Yes... I assume the writer is using Environment.NewLine?